airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcProperties.java

```diff
@@ -68,6 +68,11 @@ private static Properties commonProperties(final JdbcDatabase database) {
             : HEARTBEAT_INTERVAL;
     props.setProperty("heartbeat.interval.ms", Long.toString(heartbeatInterval.toMillis()));
 
+    if (sourceConfig.get("replication_method").has("heartbeat_action_query")
+        && !sourceConfig.get("replication_method").get("heartbeat_action_query").asText().isEmpty()) {
+      props.setProperty("heartbeat.action.query", sourceConfig.get("replication_method").get("heartbeat_action_query").asText());
```

Comment:
This seems straightforward configuring an existing debezium property.
I'm just concerned we're opening a door to all sorts of direct sql manipulation.
Not sure how but would be safer to solve it in some templated way so users or some bad actor cannot inject something.

Reply:
If most of all customers will end up configuring
`INSERT INTO airbyte_heartbeat (text) VALUES ('heartbeat')`

Can we maybe change field to input something like [schema].[table] and have our code build a query?
could be reasons not to

Reply:
Discussed offline - I'd like to not make things harder than they are by forcing users to create a specific table. Thinking here is they could just reuse one they already have instead of creating one. WDYT?